kuma/wiki/helpers.py

```diff
@@ -232,3 +233,24 @@ def wiki_url(path):
     Look for a wiki page in the current locale, or default to given path
     """
     return reverse('wiki.document', args=[path])
+
+
+def url_list(urls):
+    js_list = '[' + ', '.join('"%s"' % url for url in urls) + ']'
+    return jinja2.Markup(js_list)
```

Comment:
Ah. Makes sense now. This is building a JavaScript array for use in `config.html`. That makes sense. I think the names tripped me up a bit, partly because the term array is more conventional in JS-land. Maybe `def javascript_asset_array` and `js_array` or similar?